### ISSUE TITLE
Results from a trained model of mine.

### ISSUE THREAD

**Issue by changjo** [milesial/Pytorch-UNet#18]
Hi,
Thanks for your codes.
I trained the model with only changing batch size to 4 and I predicted some test images.
The right part in the image from a trained model is distorted and right parts in other results are also similar.
I was wondering why this happened and what your pre-trained parameters or any pre-processing except splitting a image to left and right are. Thanks.

- "Original image"
![0004d4463b50_03](https://user-images.githubusercontent.com/4277500/42427468-8eeb3ee8-8369-11e8-9dbc-92020a654508.jpg)

- Result from pre-trained model "MODEL.pth"
![pretrained_output_03](https://user-images.githubusercontent.com/4277500/42427472-9204fd3a-8369-11e8-94bf-dcb14bfe0f32.jpg)

- Result from my trained model
![trained_output_03](https://user-images.githubusercontent.com/4277500/42427471-91d79a02-8369-11e8-8183-1510ca414961.jpg)

**Comment by bermudezarii** [comment]
At the end, what did you do @changjo? Did you get a better result? Thanks! 
Because I have the same problem as yours, also, where does the model from the new training is saved? Is the last checkpoint? 

Thanks!

**Comment by changjo** [comment]
@bermudezarii I tried to change the learning rate, the training epochs, and batch size but I did not work... Currently, I have not been looking into the codes...

**Comment by LinglanZhao** [comment]
How can I get the pre-trained model "MODEL.pth"? (or I have to train a segmentation model from scratch?) @changjo @milesial

**Comment by milesial** [comment]
The pretrained model was available for the commit https://github.com/milesial/Pytorch-UNet/tree/e2e46ce509382a45b1db4e1f639aeed568f6cb3e and before. After that, I cleaned up the code and the model and I don't think the same file will load.

**Comment by LinglanZhao** [comment]
Thanks and MODEL.pth can be loaded successfully using the corresponding commit!

**Comment by changjo** [comment]
I have not recognized the repo was updated.
